templates/release-drafter.yml.jinja

```
@@ -14,7 +14,7 @@ categories:
   - title: Bug Fixes
     label: "🛠 goal: fix"
 # Identical to the default except it uses `-` instead of `*` to match our Markdown linter
-change-template: "- $TITLE (#$NUMBER) @$AUTHOR"
+change-template: "- [$TITLE](https://github.com/WordPress/openverse/pull/$NUMBER) by @$AUTHOR"
```

It might be nice to still include the `$NUMBER` in the title, since we were displaying that before! What do you think?
```suggestion
change-template: "- [$TITLE (#$NUMBER)](https://github.com/WordPress/openverse/pull/$NUMBER) by @$AUTHOR"
```

---

Sure!

---

Although, I'll need to test locally if this actually still causes the `link_issues` plugin to try to create a link on the reference :open_mouth:

---

`link_issues` skips anything that's already inside of a "reference" node, which includes links, so this should be fine! Added in https://github.com/WordPress/openverse/pull/1973/commits/7d3e374c68edefa92c2333bacd2c7d7840c4613e along with the downstream template changes.